src/dbus_fast/constants.py

```diff
@@ -19,7 +19,13 @@ class MessageType(Enum):
     SIGNAL = 4  #: A broadcast signal to subscribed connections
 
 
-MESSAGE_TYPE_MAP = {field.value: field for field in MessageType}
+# This is written out because of https://github.com/python/cpython/issues/98976
```

Comment:
Only flags are effected by the change in [cpython #98976](https://github.com/python/cpython/issues/98976) .  The change to `MESSAGE_TYPE_MAP` is unnecessary.

Reply:
What are the `MESSAGE_*_MAP` dicts used for?

Reply:
Thanks